src/crimson/osd/pg_backend.h

```diff
@@ -58,6 +58,9 @@ class PGBackend
   using interruptible_future =
     ::crimson::interruptible::interruptible_future<
       ::crimson::osd::IOInterruptCondition, T>;
+  using rep_op_fut_t =
+    std::tuple<interruptible_future<>,
+	       interruptible_future<crimson::osd::acked_peers_t>>;
```

Comment:
there are two rep_op_fut_t definition in pg.h and pg_backend.h.  can we use different name to identify them?

Reply:
ok, i'll deal with it